app/config.ts

```diff
@@ -23,7 +24,7 @@ const fallbackConfig: Config = {
   MAP_ZOOM: parseInt(process.env.MAP_ZOOM || '4', 10),
   MAP_MAX_ZOOM: parseInt(process.env.MAP_MAX_ZOOM || '13', 10),
   TWITTER: process.env.TWITTER || 'coronastatusNO',
-  ZIP_LENGTH: parseInt(process.env.ZIP_LENGTH || '4', 10),
+  ZIP_PATTERN: process.env.ZIP_PATTERN || '[A-Za-z0-9]{4}',
```

Comment:
Should we default to allow letters? I guess this won't really be a problem – but could be helpful with a comment on why we would fallback to this (as opposed to just `[0-9]{4}` :)

Reply:
I changed it to `[A-Za-z0-9]{2,10}` 😅 If someone forgets to set the pattern, I guess it is better to have very little validation.